### ISSUE TITLE
v0.5 crashes on Galaxy Note 1 (n7000)

### ISSUE THREAD

**Issue by dnnkeeper** [googlesamples/cardboard-unity#43]
I've tried building DemoScene from v0.5 package and run it on my Galaxy Note but it crashed. It works on my Galaxy Note 2 (n7100) but not on my older phones. Then I reverted to v0.4 and it worked fine there.

Looks like v0.5 libvrunity.so crashes.

Logcat:
~~~
D/dalvikvm(17148): Trying to load lib /data/data/com.Company.StereoMobile/lib/libmain.so 0x425461a0

D/dalvikvm(17148): Added shared lib /data/data/com.Company.StereoMobile/lib/libmain.so 0x425461a0

I/Unity   (17148): splash_mode = 0 (integer)

I/Unity   (17148): hide_status_bar = True (bool)

I/Unity   (17148): useObb = False (bool)

D/Unity   (17148): surfaces: attach glview

I/Unity   (17148): Choreographer available: Enabling VSYNC timing

I/Unity   (17148): onResume

D/Unity   (17148): [EGL] Attaching window :0x5b088728

D/Unity   (17148): [EGL] Attaching window :0x5b088728

I/Unity   (17148): windowFocusChanged: true

W/ActivityManager( 2015): mDVFSLock.release()

D/Unity   (17148): SystemInfo CPU = ARMv7 VFPv3 NEON, Cores = 2, Memory = 862mb

D/Unity   (17148): [SHA1] ed9813132770b56693932176ff28c299faaf71d0   20904397 bytes ~ lib/armeabi-v7a/libunity.so

D/Unity   (17148): Mono path[0] = '/data/app/com.Company.StereoMobile-2.apk/assets/bin/Data/Managed'

D/Unity   (17148): Mono config path = 'assets/bin/Data/Managed'

D/Unity   (17148): PlayerConnection initialized from /data/app/com.Company.StereoMobile-2.apk/assets/bin/Data (debug = 0)

D/Unity   (17148): PlayerConnection initialized network socket : 0.0.0.0 55064

D/Unity   (17148): PlayerConnection initialized unix socket : Unity-com.Company.StereoMobile

D/Unity   (17148): Multi-casting "[IP] 192.168.1.105 [Port] 55064 [Flags] 3 [Guid] 648366485 [EditorId] 3384970603 [Version] 1048832 [Id] AndroidPlayer(samsung_GT-N7000@192.168.1.105) [Debug] 1" to [225.0.0.222:54997]...

D/Unity   (17148): Waiting for connection from host on [192.168.1.105:55064]...

D/dalvikvm( 2708): GC_CONCURRENT freed 671K, 13% free 14593K/16583K, paused 16ms+7ms, total 53ms

D/Unity   (17148): PlayerConnection accepted from [192.168.1.100] handle:0x32

D/Unity   (17148): Using monoOptions --debugger-agent=transport=dt_socket,embedding=1,defer=y,address=0.0.0.0:56485

I/ActivityManager( 2015): Config changed: {Rect(0, 624 - 800, 656) 1 0 1.0 0mcc0mnc ru_RU sw400dp w640dp h375dp nrml land finger -keyb/v/h -nav/h s.49}

I/Unity   (17148): onConfigurationChanged

D/Unity   (17148): [EGL] Attaching window :0x5b088728

D/Unity   (17148): InitializeMonoFromMain OK (5efbaee0)

D/Unity   (17148): PlayerConnection already initialized - listening to [192.168.1.105:55064]

D/Unity   (17148): PlayerInitEngineNoGraphics OK

D/Unity   (17148): [EGL] Request: ES 2.0 RGBA32 8888 0/0

D/Unity   (17148): [EGL] Selected: ID[30] ES 2.0 RGBA32 8888 0/0

D/Unity   (17148): ANativeWindow: (1280/800) RequestedResolution: (0/0) EGLSurface: (1280/800)

D/Unity   (17148): Renderer: Mali-400 MP

D/Unity   (17148): Vendor:   ARM

D/Unity   (17148): Version:  OpenGL ES 2.0

D/Unity   (17148): GLES:     2

D/Unity   (17148): GL_EXT_debug_marker GL_OES_texture_npot GL_OES_compressed_ETC1_RGB8_texture GL_OES_standard_derivatives GL_OES_EGL_image GL_OES_depth24 GL_ARM_rgba8 GL_ARM_mali_shader_binary GL_OES_depth_texture GL_OES_packed_depth_stencil GL_EXT_texture_format_BGRA8888 GL_EXT_blend_minmax GL_OES_EGL_image_external GL_OES_EGL_sync GL_OES_rgb8_rgba8 GL_EXT_multisampled_render_to_texture GL_EXT_discard_framebuffer 

D/Unity   (17148): OPENGL LOG: Creating OpenGL ES 2.0 graphics device

D/Unity   (17148): InitializeGfxDevice OK

D/Unity   (17148): Initialize engine version: 5.1.1f1 (2046fc06d4d8)

D/Unity   (17148): Begin MonoManager ReloadAssembly

D/Unity   (17148): Platform assembly: /data/app/com.Company.StereoMobile-2.apk/assets/bin/Data/Managed/UnityEngine.dll (this message is harmless)

D/Unity   (17148): Loading /data/app/com.Company.StereoMobile-2.apk/assets/bin/Data/Managed/UnityEngine.dll into Unity Child Domain

D/Unity   (17148): Platform assembly: /data/app/com.Company.StereoMobile-2.apk/assets/bin/Data/Managed/Assembly-CSharp-firstpass.dll (this message is harmless)

D/Unity   (17148): Loading /data/app/com.Company.StereoMobile-2.apk/assets/bin/Data/Managed/Assembly-CSharp-firstpass.dll into Unity Child Domain

D/Unity   (17148): Platform assembly: /data/app/com.Company.StereoMobile-2.apk/assets/bin/Data/Managed/Assembly-CSharp.dll (this message is harmless)

D/Unity   (17148): Loading /data/app/com.Company.StereoMobile-2.apk/assets/bin/Data/Managed/Assembly-CSharp.dll into Unity Child Domain

D/Unity   (17148): Platform assembly: /data/app/com.Company.StereoMobile-2.apk/assets/bin/Data/Managed/Assembly-UnityScript.dll (this message is harmless)

D/Unity   (17148): Loading /data/app/com.Company.StereoMobile-2.apk/assets/bin/Data/Managed/Assembly-UnityScript.dll into Unity Child Domain

D/Unity   (17148): Platform assembly: /data/app/com.Company.StereoMobile-2.apk/assets/bin/Data/Managed/UnityEngine.UI.dll (this message is harmless)

D/Unity   (17148): Loading /data/app/com.Company.StereoMobile-2.apk/assets/bin/Data/Managed/UnityEngine.UI.dll into Unity Child Domain

D/Unity   (17148): Platform assembly: /data/app/com.Company.StereoMobile-2.apk/assets/bin/Data/Managed/UnityEngine.Networking.dll (this message is harmless)

D/Unity   (17148): Loading /data/app/com.Company.StereoMobile-2.apk/assets/bin/Data/Managed/UnityEngine.Networking.dll into Unity Child Domain

D/Unity   (17148): Platform assembly: /data/app/com.Company.StereoMobile-2.apk/assets/bin/Data/Managed/UnityEngine.Analytics.dll (this message is harmless)

D/Unity   (17148): Loading /data/app/com.Company.StereoMobile-2.apk/assets/bin/Data/Managed/UnityEngine.Analytics.dll into Unity Child Domain

D/Unity   (17148): - Completed reload, in  0.198 seconds

D/Unity   (17148): PlayerInitEngineGraphics OK

D/Unity   (17148): Found 11 native sensors

D/Unity   (17148): Sensor :        Accelerometer ( 1) ; 0.004788 / 0.00s ; K3DH Acceleration Sensor / STMicroelectronics 

D/Unity   (17148): Sensor :        Accelerometer ( 1) ; 0.004788 / 0.00s ; K3DH Acceleration Sensor / STMicroelectronics 

D/Unity   (17148): Requested framebuffer: resolution[1280x800], rgba[8/8/8/8], depth+stencil[on], samples[1]

D/Unity   (17148): Created framebuffer: resolution[1280x800], rgba[8/8/8/8], depth+stencil[24/8], samples[0] 

D/Unity   (17148): Platform assembly: /data/app/com.Company.StereoMobile-2.apk/assets/bin/Data/Managed/System.Core.dll (this message is harmless)

D/Unity   (17148): Platform assembly: /data/app/com.Company.StereoMobile-2.apk/assets/bin/Data/Managed/System.dll (this message is harmless)

D/dalvikvm( 2015): WAIT_FOR_CONCURRENT_GC blocked 0ms

D/dalvikvm( 2015): GC_EXPLICIT freed 5827K, 50% free 33181K/66119K, paused 4ms+12ms, total 112ms

W/Unity   (17148): The referenced script on this Behaviour is missing!

W/Unity   (17148):  

W/Unity   (17148): (Filename: ./Runtime/Mono/MonoBehaviour.cpp Line: 1715)

W/Unity   (17148): 

W/Unity   (17148): The referenced script on this Behaviour is missing!

W/Unity   (17148):  

W/Unity   (17148): (Filename: ./Runtime/Mono/MonoBehaviour.cpp Line: 1715)

W/Unity   (17148): 

W/Unity   (17148): The referenced script on this Behaviour is missing!

W/Unity   (17148):  

W/Unity   (17148): (Filename: ./Runtime/Mono/MonoBehaviour.cpp Line: 1715)

W/Unity   (17148): 

W/Unity   (17148): The referenced script on this Behaviour is missing!

W/Unity   (17148):  

W/Unity   (17148): (Filename: ./Runtime/Mono/MonoBehaviour.cpp Line: 1715)

W/Unity   (17148): 

I/Unity   (17148): Default clip could not be found in attached animations list.

I/Unity   (17148):  

I/Unity   (17148): (Filename: ./Runtime/Animation/Animation.cpp Line: 400)

I/Unity   (17148): 

D/dalvikvm(17148): Trying to load lib /data/data/com.Company.StereoMobile/lib/libvrunity.so 0x425461a0

D/dalvikvm(17148): Added shared lib /data/data/com.Company.StereoMobile/lib/libvrunity.so 0x425461a0

I/DEBUG   ( 1715): *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***

I/DEBUG   ( 1715): Build fingerprint: 'samsung/GT-N7000/GT-N7000:4.1.2/JZO54K/N7000XXLSZ:user/release-keys'

I/DEBUG   ( 1715): pid: 17148, tid: 17169, name: UnityMain  >>> com.Company.StereoMobile <<<

I/DEBUG   ( 1715): signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 00000000

I/ActivityManager( 2015): Process com.google.android.talk (pid 16209) (adj 11) has died.

I/ActivityManager( 2015): Process com.sec.chaton (pid 15906) (adj 12) has died.

I/ActivityManager( 2015): Process com.google.android.music:main (pid 16506) (adj 11) has died.

I/DEBUG   ( 1715):     r0 41644970  r1 00000000  r2 41644120  r3 00000000

I/DEBUG   ( 1715):     r4 00000000  r5 5e7f22e4  r6 00000000  r7 670eb20c

I/DEBUG   ( 1715):     r8 00000000  r9 670deb1c  sl 5e7f2310  fp 5e7f22e0

I/DEBUG   ( 1715):     ip 41644990  sp 5e7f2298  lr 4086cafb  pc 4086cafe  cpsr 60000030

I/DEBUG   ( 1715):     d0  00000140439c0de3  d1  3f80000042700000

I/DEBUG   ( 1715):     d2  407381bc60000000  d3  407381bc60000000

I/DEBUG   ( 1715):     d4  0000000000000000  d5  000001f600000000

I/DEBUG   ( 1715):     d6  439c0de300000000  d7  38aaab9b38aaab9b

I/DEBUG   ( 1715):     d8  0000000000000000  d9  0000000000000000

I/DEBUG   ( 1715):     d10 0000000000000000  d11 0000000000000000

I/DEBUG   ( 1715):     d12 0000000000000000  d13 0000000000000000

I/DEBUG   ( 1715):     d14 0000000000000000  d15 0000000000000000

I/DEBUG   ( 1715):     d16 0000000300000000  d17 0000050000000001

I/DEBUG   ( 1715):     d18 0000050000000320  d19 439c0de300000320

I/DEBUG   ( 1715):     d20 61c1aa0042640106  d21 3f80000000000000

I/DEBUG   ( 1715):     d22 0000000000000000  d23 3f80000000000000

I/DEBUG   ( 1715):     d24 3f7fec7cbc8da925  d25 000000003c8d07a1

I/DEBUG   ( 1715):     d26 bc8b3080bf7fec7c  d27 00000000bc8f7800

I/DEBUG   ( 1715):     d28 bc8f7800bc8d0790  d29 000000003f7fec3c

I/DEBUG   ( 1715):     d30 3f238c40bbed0000  d31 3f8000003b990000

I/DEBUG   ( 1715):     scr 8000001a

I/DEBUG   ( 1715): 

I/DEBUG   ( 1715): backtrace:

I/DEBUG   ( 1715):     #00  pc 0004cafe  /system/lib/libdvm.so

I/DEBUG   ( 1715):     #01  pc 00173983  /data/data/com.Company.StereoMobile/lib/libvrunity.so (gvr::PlatformInfo::PlatformInfo()+626)

I/DEBUG   ( 1715): 

I/DEBUG   ( 1715): stack:

I/DEBUG   ( 1715):          5e7f2258  00000000  

I/DEBUG   ( 1715):          5e7f225c  67082368  /data/data/com.Company.StereoMobile/lib/libvrunity.so

I/DEBUG   ( 1715):          5e7f2260  41a1f800  /dev/ashmem/dalvik-heap (deleted)

I/DEBUG   ( 1715):          5e7f2264  670e06a0  /data/data/com.Company.StereoMobile/lib/libvrunity.so

I/DEBUG   ( 1715):          5e7f2268  40869d07  /system/lib/libdvm.so

I/DEBUG   ( 1715):          5e7f226c  00000000  

I/DEBUG   ( 1715):          5e7f2270  5e7f229c  

I/DEBUG   ( 1715):          5e7f2274  5e7f22e4  

I/DEBUG   ( 1715):          5e7f2278  00000000  

I/DEBUG   ( 1715):          5e7f227c  40868f8d  /system/lib/libdvm.so

I/DEBUG   ( 1715):          5e7f2280  5e7f229c  

I/DEBUG   ( 1715):          5e7f2284  41644120  

I/DEBUG   ( 1715):          5e7f2288  00000000  

I/DEBUG   ( 1715):          5e7f228c  5e7f22e4  

I/DEBUG   ( 1715):          5e7f2290  df0027ad  

I/DEBUG   ( 1715):          5e7f2294  00000000  

I/DEBUG   ( 1715):     #00  5e7f2298  41644120  

I/DEBUG   ( 1715):          5e7f229c  41644970  

I/DEBUG   ( 1715):          5e7f22a0  41644120  

I/DEBUG   ( 1715):          5e7f22a4  67032985  /data/data/com.Company.StereoMobile/lib/libvrunity.so (gvr::PlatformInfo::PlatformInfo()+628)

I/DEBUG   ( 1715):     #01  5e7f22a8  6681c510  

I/DEBUG   ( 1715):          5e7f22ac  5d4b8c1c  /data/data/com.Company.StereoMobile/lib/libmono.so

I/DEBUG   ( 1715):          5e7f22b0  00000004  

I/DEBUG   ( 1715):          5e7f22b4  5e7f2304  

I/DEBUG   ( 1715):          5e7f22b8  5e7f23c8  

I/DEBUG   ( 1715):          5e7f22bc  00000010  

I/DEBUG   ( 1715):          5e7f22c0  670deb30  /data/data/com.Company.StereoMobile/lib/libvrunity.so

I/DEBUG   ( 1715):          5e7f22c4  670d9d50  /data/data/com.Company.StereoMobile/lib/libvrunity.so

I/DEBUG   ( 1715):          5e7f22c8  670da038  /data/data/com.Company.StereoMobile/lib/libvrunity.so

I/DEBUG   ( 1715):          5e7f22cc  670deab0  /data/data/com.Company.StereoMobile/lib/libvrunity.so

I/DEBUG   ( 1715):          5e7f22d0  670d9fc8  /data/data/com.Company.StereoMobile/lib/libvrunity.so

I/DEBUG   ( 1715):          5e7f22d4  670d9fb4  /data/data/com.Company.StereoMobile/lib/libvrunity.so

I/DEBUG   ( 1715):          5e7f22d8  6681c510  

I/DEBUG   ( 1715):          5e7f22dc  00000000  

I/DEBUG   ( 1715):          5e7f22e0  670eb218  

I/DEBUG   ( 1715):          5e7f22e4  670eb218  

I/DEBUG   ( 1715): 

I/DEBUG   ( 1715): memory near r0:

I/DEBUG   ( 1715):     41644950 00000000 00000000 00000000 00000000  ................

I/DEBUG   ( 1715):     41644960 00000000 00000000 72745374 00000453  ........tStrS...

I/DEBUG   ( 1715):     41644970 5d0fb7a0 5a665fac 573f2028 5b08f000  ...]._fZ( ?W...[

I/DEBUG   ( 1715):     41644980 00000000 00000000 5e7f2da8 00000000  .........-.^....

I/DEBUG   ( 1715):     41644990 5e7f2ddc 0000000c 00000000 4083e0c0  .-.^...........@

I/DEBUG   ( 1715): 

I/DEBUG   ( 1715): memory near r2:

I/DEBUG   ( 1715):     41644100 65726f68 6172676f 72656870 6c614324  horeographer$Cal

I/DEBUG   ( 1715):     41644110 6361626c 6575516b 003b6575 00000023  lbackQueue;.#...

I/DEBUG   ( 1715):     41644120 408c3dd4 00000000 0000000c 41644970  .=.@........pIdA

I/DEBUG   ( 1715):     41644130 00000000 5b08b440 4163bb90 00000023  ....@..[..cA#...

I/DEBUG   ( 1715):     41644140 425461a0 4007e544 2e643379 79616c70  .aTBD..@y3d.play

I/DEBUG   ( 1715): 

I/DEBUG   ( 1715): memory near r5:

I/DEBUG   ( 1715):     5e7f22c4 670d9d50 670da038 670deab0 670d9fc8  P..g8..g...g...g

I/DEBUG   ( 1715):     5e7f22d4 670d9fb4 6681c510 00000000 670eb218  ...g...f.......g

I/DEBUG   ( 1715):     5e7f22e4 670eb218 670da040 6681f8a5 6681f8a5  ...g@..g...f...f

I/DEBUG   ( 1715):     5e7f22f4 6681f8a5 6681f8a4 6681f8a6 6681faa4  ...f...f...f...f

I/DEBUG   ( 1715):     5e7f2304 670eaa30 00000010 6681f8a4 670de760  0..g.......f`..g

I/DEBUG   ( 1715): 

I/DEBUG   ( 1715): memory near r7:

I/DEBUG   ( 1715):     670eb1ec 00000000 00000000 00000000 00000000  ................

I/DEBUG   ( 1715):     670eb1fc 00000000 00000000 00000000 00000000  ................

I/DEBUG   ( 1715):     670eb20c 00000000 00000000 00000000 00000000  ................

I/DEBUG   ( 1715):     670eb21c 00000000 00000000 00000000 00000000  ................

I/DEBUG   ( 1715):     670eb22c 00000000 00000000 00000002 00000010  ................

I/DEBUG   ( 1715): 

I/DEBUG   ( 1715): memory near r9:

I/DEBUG   ( 1715):     670deafc ffffffd0 ffffffd0 670de788 00000000  ...........g....

I/DEBUG   ( 1715):     670deb0c 00000000 0000002c 00000000 670dea78  ....,.......x..g

I/DEBUG   ( 1715):     670deb1c 00000000 00000000 ffffffd4 ffffffd4  ................

I/DEBUG   ( 1715):     670deb2c 670dea78 00000000 00000000 0000002c  x..g........,...

I/DEBUG   ( 1715):     670deb3c 00000000 670dea78 00000000 00000000  ....x..g........

I/DEBUG   ( 1715): 

I/DEBUG   ( 1715): memory near sl:

I/DEBUG   ( 1715):     5e7f22f0 6681f8a5 6681f8a5 6681f8a4 6681f8a6  ...f...f...f...f

I/DEBUG   ( 1715):     5e7f2300 6681faa4 670eaa30 00000010 6681f8a4  ...f0..g.......f

I/DEBUG   ( 1715):     5e7f2310 670de760 00000006 00000000 00001002  `..g............

I/DEBUG   ( 1715):     5e7f2320 00000000 00000000 00000000 00000000  ................

I/DEBUG   ( 1715):     5e7f2330 00000000 00000000 00000000 00000000  ................

I/DEBUG   ( 1715): 

I/DEBUG   ( 1715): memory near fp:

I/DEBUG   ( 1715):     5e7f22c0 670deb30 670d9d50 670da038 670deab0  0..gP..g8..g...g

I/DEBUG   ( 1715):     5e7f22d0 670d9fc8 670d9fb4 6681c510 00000000  ...g...g...f....

I/DEBUG   ( 1715):     5e7f22e0 670eb218 670eb218 670da040 6681f8a5  ...g...g@..g...f

I/DEBUG   ( 1715):     5e7f22f0 6681f8a5 6681f8a5 6681f8a4 6681f8a6  ...f...f...f...f

I/DEBUG   ( 1715):     5e7f2300 6681faa4 670eaa30 00000010 6681f8a4  ...f0..g.......f

I/DEBUG   ( 1715): 

I/DEBUG   ( 1715): memory near ip:

I/DEBUG   ( 1715):     41644970 5d0fb7a0 5a665fac 573f2028 5b08f000  ...]._fZ( ?W...[

I/DEBUG   ( 1715):     41644980 00000000 00000000 5e7f2da8 00000000  .........-.^....

I/DEBUG   ( 1715):     41644990 5e7f2ddc 0000000c 00000000 4083e0c0  .-.^...........@

I/DEBUG   ( 1715):     416449a0 00000000 00000000 5667e170 5a662300  ........p.gV.#fZ

I/DEBUG   ( 1715):     416449b0 00000000 00000000 00000001 00004000  .............@..

I/DEBUG   ( 1715): 

I/DEBUG   ( 1715): memory near sp:

I/DEBUG   ( 1715):     5e7f2278 00000000 40868f8d 5e7f229c 41644120  .......@.".^ AdA

I/DEBUG   ( 1715):     5e7f2288 00000000 5e7f22e4 df0027ad 00000000  .....".^.'......

I/DEBUG   ( 1715):     5e7f2298 41644120 41644970 41644120 67032985   AdApIdA AdA.).g

I/DEBUG   ( 1715):     5e7f22a8 6681c510 5d4b8c1c 00000004 5e7f2304  ...f..K].....#.^

I/DEBUG   ( 1715):     5e7f22b8 5e7f23c8 00000010 670deb30 670d9d50  .#.^....0..gP..g

I/DEBUG   ( 1715): 

I/DEBUG   ( 1715): code around pc:

I/DEBUG   ( 1715):     4086cadc fa5ef7fc bd3e4620 b5134602 a801460c  ..^. F>..F...F..

I/DEBUG   ( 1715):     4086caec f7fc4611 4621fa31 f7fd9801 4603fa29  .F..1.!F....)..F

I/DEBUG   ( 1715):     4086cafc 68199801 fff2f7fb a8014604 fa48f7fc  ...h.....F....H.

I/DEBUG   ( 1715):     4086cb0c bd1c4620 4606b5f0 460cb087 4631a803   F.....F...F..1F

I/DEBUG   ( 1715):     4086cb1c 461f4615 fa18f7fc 98034621 fa10f7fd  .F.F....!F......

I/DEBUG   ( 1715): 

I/DEBUG   ( 1715): code around lr:

I/DEBUG   ( 1715):     4086cad8 a8012401 fa5ef7fc bd3e4620 b5134602  .$....^. F>..F..

I/DEBUG   ( 1715):     4086cae8 a801460c f7fc4611 4621fa31 f7fd9801  .F...F..1.!F....

I/DEBUG   ( 1715):     4086caf8 4603fa29 68199801 fff2f7fb a8014604  )..F...h.....F..

I/DEBUG   ( 1715):     4086cb08 fa48f7fc bd1c4620 4606b5f0 460cb087  ..H. F.....F...F

I/DEBUG   ( 1715):     4086cb18 4631a803 461f4615 fa18f7fc 98034621  ..1F.F.F....!F..

I/DEBUG   ( 1715): !@dumpstate -k -t -z -d -o /data/log/dumpstate_app_native -m 17148

I/ActivityManager( 2015): Process com.Company.StereoMobile (pid 17148) (adj 0) has died.

I/ActivityManager( 2015): Config changed: {Rect(624, 0 - 656, 800) 1 0 1.0 0mcc0mnc ru_RU sw400dp w400dp h615dp nrml port finger -keyb/v/h -nav/h s.50}

D/dalvikvm( 2165): GC_CONCURRENT freed 1882K, 14% free 23271K/26951K, paused 21ms+45ms, total 266ms

D/dalvikvm( 2337): WAIT_FOR_CONCURRENT_GC blocked 0ms

D/dalvikvm(17014): GC_CONCURRENT freed 339K, 10% free 12231K/13511K, paused 10ms+2ms, total 99ms

D/dalvikvm( 2337): GC_EXPLICIT freed 3293K, 12% free 39400K/44679K, paused 35ms+16ms, total 193ms

D/dalvikvm( 2015): WAIT_FOR_CONCURRENT_GC blocked 0ms

D/dalvikvm( 2015): GC_EXPLICIT freed 2222K, 51% free 32815K/66119K, paused 187ms+10ms, total 416ms


~~~

**Comment by Nitro888** [comment]
May be same problem with me. fix it please.

**Comment by smdol** [comment]
That bug has been logged internally.  A fix will be forthcoming.

**Comment by aviomaksim** [comment]
smdol, could you say when fix will come? We are thinking to downgrade, because of this issue.

**Comment by smdol** [comment]
Unfortunately I'm really not allowed to give specific dates or time frames.  Sorry...

**Comment by aviomaksim** [comment]
Thank you. Problem is solved at v0.5.1. :+1:
